HelpSource/Tutorials/Getting-Started/04-Functions-and-Other-Functionality.schelp

```diff
@@ -164,7 +205,24 @@ myFunc.value("bar");
 myFunc;			// throws an error
 ::
 
-You may be wondering why we haven't needed to declare variables like code::f::, and why they don't seem to have any particular scope (i.e. they keep their values even when executing code one line at a time). The letters a to z are what are called interpreter variables. These are pre-declared when you start up SC, and have an unlimited, or 'global', scope. This makes them useful for quick tests or examples. You've already encountered one of these, the variable 's', which you'll recall by default refers to the localhost server.
+You may be wondering why we haven't needed to declare variables like code::f::, and why they don't seem to have any particular scope (i.e. they keep their values even when executing code one line at a time). The letters a to z are what are called interpreter variables. These are pre-declared when you start up SC, and have an unlimited, or "global", scope. This makes them useful for quick tests or examples. You've already encountered one of these, the variable "s", which you'll recall by default refers to the localhost server.
+
+With arguments, we can see an example of how polymorphism works:
+
+code::
+(
+f = {
+  arg a;
+  a.value + 3;       // call "value" on a; polymorphism awaits!
+};
+)
+f.value(3);          // a.value is 3, so this returns 3 + 3 = 6
+g = { 3.0.rand; };
```

Comment:
`g = { 3.0.rand }` 

(better leave out semicolons for the last statement in a function)